services/web/server/src/simcore_service_webserver/projects/_handlers_project_comments.py

```diff
@@ -0,0 +1,294 @@
+""" Handlers for project comments operations
+
+"""
+
+import logging
+from typing import Any
+
+from aiohttp import web
+from models_library.projects import ProjectID
+from models_library.projects_comments import CommentID
+from models_library.rest_pagination import DEFAULT_NUMBER_OF_ITEMS_PER_PAGE, Page
+from models_library.rest_pagination_utils import paginate_data
+from models_library.users import UserID
+from pydantic import BaseModel, Extra, Field, NonNegativeInt
+from servicelib.aiohttp.requests_validation import (
+    parse_request_body_as,
+    parse_request_path_parameters_as,
+    parse_request_query_parameters_as,
+)
+from servicelib.json_serialization import json_dumps
+from servicelib.mimetype_constants import MIMETYPE_APPLICATION_JSON
+from servicelib.rest_constants import RESPONSE_MODEL_POLICY
+
+from .._meta import api_version_prefix as VTAG
+from ..login.decorators import login_required
+from ..security.decorators import permission_required
+from . import projects_api
+from ._handlers_crud import RequestContext
+from .exceptions import ProjectNotFoundError
+
+_logger = logging.getLogger(__name__)
+
+
+routes = web.RouteTableDef()
+
+
+class _CreateProjectCommentsPathParams(BaseModel):
+    project_uuid: ProjectID
```

Comment:
MINOR: since all routes under projects start with ``projects/{project_id}`` ... perhaps we could add a common `*PathParms` validation class?